frontend/src/v5/services/api/containers.ts

```diff
@@ -17,24 +17,49 @@
 
 import { AxiosResponse } from 'axios';
 import {
-	DeleteContainerPayload,
-	FavouritePayload,
-	FetchContainersPayload, FetchContainersResponse,
-	FetchContainerStatsPayload, FetchContainerStatsResponse,
-	CreateContainerPayload,
+	ContainerStats,
+	NewContainer,
 } from '@/v5/store/containers/containers.types';
+import { TeamspaceAndProjectId, TeamspaceProjectAndContainerId } from '@/v5/store/store.types';
 import api from './default';
 
+/**
+ * Types
+*/
+type FetchContainersParams = TeamspaceAndProjectId;
```

Comment:
What's the main advantage of renaming these types instead of just using them (e.g. TeamspaceAndProjectId) directly in the redux functions below?

Reply:
No reason to be honest!